_posts/en/2016-xx-xx-assignment-shorthands.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+
+title: Assignment Shorthands
+tip-number: 25
+tip-username: hsleonis
+tip-username-profile: https://github.com/hsleonis
+tip-tldr: Simple shorthands to make code cleaner and simpler.
+
+categories:
+    - en
+---
+
+## Assignment Shorthands
```

Comment:
Title is added automatically to the post, so this post will have two same titles